ads/smartadserver.md

```diff
@@ -16,14 +16,35 @@ limitations under the License.
 
 # SmartAdServer
 
+## Supported Smart AdServer parameters in the amp-ad tag
+
+All of the parameters listed here should be prefixed with "data-" when used.
+
+| Parameter name | Description                         | Required |
+|----------------|-------------------------------------|----------|
+| site           | Your Smart AdServer Site ID         | Yes      |
+| page           | Your Smart AdServer Page ID         | Yes      |
+| format         | Your Smart AdServer Format ID       | Yes      |
+| domain         | Your Smart AdServer call domain     | Yes      |
+| target         | Your targeting string               | No       |
+| tag            | An ID for the tag containing the ad | No       |
+
+Note: If any of the required parameters is missing, the integration won't be able to work correctly,
+the Smart AdServer integration will call the ```window.context.noContentAvailable()``` function
```

Comment:
Sorry for not stating this clearly in the first place. 

This audience of this doc is publisher who uses SmartAdServer, I think there's no need to mention the implementation details about noContentAvailable and renderStart.